rhcos-layer/README.md

```diff
@@ -0,0 +1,88 @@
+# Creating RHCOS image Layer
+
+This folder contains artifacts to create RHCOS image layer for SNP and TDX.
+
+The RHCOS image layer contains the required kernel version, kata and qemu rpms.
+
+## Download the required rpms
+
+For SNP
+
+```sh
+cd rpms-download
+./download-rpms.sh rhel-rpms.yaml
+```
+
+Copy the downloaded rpms to the `snp/rpms` folder
+
+For TDX
+
+```sh
+cd rpms-download
+./download-rpms.sh centos-rpms.yaml
+```
+
+Copy the downloaded rpms to the `tdx/rpms` folder
+
+## Build the RHCOS layer
+
+### With access to OCP cluster
+
+Build for SNP
+
+```sh
+sudo make TEE=snp build
```

Comment:
First time, before calling `make build`, one should run `make get-ocp-release-image` and `make get-pull-secret` to get info about ocp release and pull secret.

Reply:
I had the dependent target commented out. Have enabled it for build